unit_test.py

```diff
@@ -1,38 +1,41 @@
 import unittest
 
 from api import GiphyAPI
-from data import Data
+
+gAPI = GiphyAPI()
 
 
 class Test(unittest.TestCase):
     # small tests
     def test_url(self):
-        api_key = "oZhufry2NyjOWfmCItORiADf8lPGTN2j"
```

Comment:
Looks like your API key is now a part of your git history. That's like committing a password to your git repository. You will need to remove it. [Here are some instructions on how to do that](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository).

Reply:
That's what I deleted for this pull request! Now the API key doesn't appear in the file.

Reply:
That's what I deleted for this pull request! Now the API key doesn't appear in the file.